lib/gds_api/support_api.rb

```diff
@@ -21,6 +21,10 @@ def create_global_export_request(request_details)
     post_json!("#{endpoint}/anonymous-feedback/global-export-requests", global_export_request: request_details)
   end
 
+  def create_page_improvement(params)
```

Comment:
Can you please document it?
Example: https://github.com/alphagov/gds-api-adapters/pull/581/files#diff-c279de60f2b72b505bbabe7401804a88R98